__tests__/api.test.js

```diff
@@ -91,4 +91,30 @@ describe("API endpoints", () => {
                 expect(articles).toBeSortedBy(expected, {descending: true})
         });
     })
+    test("/api/articles/:article_id/comments endpoint should return all article comments in an array", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+            .then((result) => {
+                const comments = result.body.comments;
+                const expected = 11
+                expect(comments.length).toBe(expected)
+                expect(comments[0]).toHaveProperty('comment_id')
+                expect(comments[0]).toHaveProperty('votes')
+                expect(comments[0]).toHaveProperty('created_at')
+                expect(comments[0]).toHaveProperty('author')
+                expect(comments[0]).toHaveProperty('body')
+                expect(comments[0]).toHaveProperty('article_id')
```

Comment:
This is only checking if the first element in the comments array contains these properties, but it should be checking that every element contains these properties. 

We also know what the value of article_id should be for each of these comments, so this should be included in the above assertion as well.